test-js/unit/pentaho/config/impl/AmdLoadedService.Spec.js

```diff
@@ -34,19 +34,25 @@ define([], function() {
         config3 = {
           rules: [
             {
-              select: {},
+              select: {
+                type: "A"
```

Comment:
An empty `type` could still mean _apply to everything_, but I'm also ok with making it mandatory.